docs/lesiaturuta/5-logic/homework.js

```diff
@@ -0,0 +1,119 @@
+/**
+ *
+ * @param firstValue string with first number
+ * @param secondValue string with second number
+ * @param operation string with operation symbol
+ * @returns {string|number} result of calculation or error message:
+ * - if firstValue or secondValue is not a number, return 'Enter a number'
+ * - if operation is not '+', '-', '*', '/', return 'Choose a valid operation'
+ * - if the result is more than 100, return 'Result is too big'
+ * - else return number of result: 1 '+' 2 returns 3
+ *
+ */
+function calculate(firstValue, secondValue, operation) {
+    const operations = ['/', '*', '-', '+'];
+
+    const firstNumber = Number(firstValue);
+    const secondNumber = Number(secondValue);
+
+    if (Number.isNaN(firstNumber)) {
```

Comment:
Should be an error for this case 
<img width="1257" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/e7606a8e-bfef-4083-ae8d-af55e4211f31">

Reply:
fixed

Reply:
Still no errors
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/6f1c81c5-edc7-4734-bf0a-0ea9e718a1ca)